subsys/emul/emul_bmi160.c

```diff
@@ -241,7 +241,7 @@ static int bmi160_emul_transfer_i2c(const struct emul *target, struct i2c_msg *m
 
 	__ASSERT_NO_MSG(msgs && num_msgs);
 
-	i2c_dump_msgs("emul", msgs, num_msgs, addr);
```

Comment:
Are you planning to deprecate `i2c_dump_msgs` later?

Reply:
Certainly could be done.  From the description of the process, it doesn't look like a mere contributor has any hope of doing it.